src/clients/node/scripts/test_install_on_alpine.sh

```diff
@@ -2,8 +2,6 @@
 
 set -e
 
-./scripts/build.sh
```

Comment:
Can we have some way of checking if the binary is here? Or maybe only run if not in the github environment?

Reply:
Done! There's a variable that can be set `SKIP_NODE_BUILD` to skip building, and CI sets it